marimo/_output/formatting.py

```diff
@@ -161,7 +161,9 @@ def f_mime(obj: T) -> tuple[KnownMimeType, str]:
         ("_repr_png_", "image/png"),
         ("_repr_jpeg_", "image/jpeg"),
         ("_repr_markdown_", "text/markdown"),
-        ("_repr_latex_", "text/latex"),
+        # Commenting out _repr_latex_ for now, since pandas pd.Series
+        # uses it for some reason, but returns empty
+        # ("_repr_latex_", "text/latex"),
```

Comment:
@mscolnick Shouldn't our registered formatter be returned for `pd.Series`? Line 127 of this method? That is, I would have thought that `get_formatter()` would have already returned a registered formatter for `pd.Series`

Reply:
yea they _should_, but i dont think they do. its possible they were not registered

Reply:
ive narrowed it down to module reloading. i think the formatter gets lots or a new module of pandas fails an equality check